# Request message types
int8 num_dominos
#int[] num_dots_half1
#int[] num_dots_half2
#float32[] x
#float32[] y
#float32[] orientation
